R/sql_load.R

```diff
@@ -1,11 +1,24 @@
 get_sql_load <- function(path) {
-  statement <- paste(readLines(path), collapse = "\n")
+  ## It is not very nice that we have to read & parse this file to send it to
+  ## DB, ideally we could just execute the file directly on DB but DBI only
+  ## allows sending individual queries atm see
+  ## https://github.com/r-dbi/RPostgres/issues/213
+  sql <- readLines(path)
+  sql <- sql[!is_empty(sql)]
+  statement <- paste(sql, collapse = "\n")
+  statements <- unlist(strsplit(statement, ";", fixed = TRUE))
```

Comment:
Having worked around this before I suggest you refactor this into one function to split your sql into chunks vs one to run it